src/__tests__/dependency-resolution.js

```diff
@@ -488,43 +462,43 @@ tape('dependency registration with circular dependency', t => {
   t.end();
 });
 
-tape('dependency configuration', t => {
-  const StringToken: string = (() => {}: any);
-  const OtherStringToken: string = (() => {}: any);
-  const NumberToken: number = (() => {}: any);
-  const ObjectToken: {|
-    a: string,
-  |} = (() => {}: any);
-
-  const app = new App('el', el => el);
-  app.register(
-    createPlugin({
-      deps: {
-        a: StringToken,
-        b: OtherStringToken,
-        c: NumberToken,
-        d: ObjectToken,
-      },
-      provides: deps => {
-        t.equal(deps.a, 'string-a');
-        t.equal(deps.b, 'string-b');
-        t.equal(deps.c, 5);
-        t.deepLooseEqual(deps.d, {a: 'some-d'});
-        t.end();
-        return {};
-      },
-    })
-  );
-  app.register(StringToken, 'string-a');
-  app.register(OtherStringToken, 'string-b');
-  app.register(NumberToken, 5);
-  app.register(ObjectToken, {a: 'some-d'});
-  app.resolve();
-});
+// tape('dependency configuration', t => {
```

Comment:
See [comment above](https://github.com/fusionjs/fusion-core/pull/124/files#diff-877e28e5e548e988a1bc394588eccc56R55) for why this is not currently supported.